examples/lora_inference.py

```diff
@@ -4,7 +4,7 @@
 
 sql_lora_path = snapshot_download(repo_id="yard1/llama-2-7b-sql-lora-test")
 
-llm = LLM(model="meta-llama/Llama-2-7b-hf", enable_lora=True, max_num_seqs=16, dtype='bfloat16')
+llm = LLM(model="meta-llama/Llama-2-7b-hf", enable_lora=True, max_num_seqs=16)
```

Comment:
@JHLEE17 Although Gaudi2 supports FP16, we use BF16 for most of our development and testing. So we would prefer to concentrate only on BF16 (unless there is a good reason for using FP16). 
Regarding Gaudi-BF16 != A100-BF16 we will check if its a real issue or not.

Reply:
@vivekgoe While the generated tokens of the Llama-2-7B model (FP16) were identical on both Gaudi and A100 devices, there were slight differences in the logit values. Additionally, for the Llama-3-8B model, each case (whether FP16/BF16 on Gaudi/A100) produced slightly different results. I suspect these inconsistency are inherent to floating point operations in generative models. I would greatly appreciate any insights or opinions you could provide on this matter.

Reply:
@JHLEE17 Slight differences in logit values between Gaudi/A100 are expected due to following reasons,
-  For operations on large tensor inputs Gaudi & A100 may slice the input tensors and combine back outputs differently, mathematically both produce same output but due to quantization errors inherent to floating point representation this will result in small differences in final outputs.  However if we use some utility like lm_harness to evaluate quality of output generated over a input dataset with multiple prompts then we do expect to see same accuracy results for Gaudi/A100. 
-  For some operations which may use random number generators (e.g. sampling), both Gaudi & A100 will choose from same distributions but actual values picked (even with same seed) on a given iteration will be different because random number generator implementations will be different.